cantools/database/can/formats/dbc.py

```diff
@@ -854,28 +854,26 @@ def get_value(attribute):
 
         return result
 
-    if database.dbc is not None and database.dbc.attributes_rel is not None:
-        attributes_rel = database.dbc.attributes_rel
-        for frame_id, element in attributes_rel.items():
-            if "signal" in element:
-                for signal_name, signal_lst in element['signal'].items():
-                    for node_name, node_dict in signal_lst['node'].items():
-                        for attribute in node_dict.values():
-                            ba_rel.append(f'BA_REL_ "{attribute.definition.name}" '
-                                          f'BU_SG_REL_ '
-                                          f'{node_name} '
-                                          f'SG_ '
-                                          f'{frame_id} '
-                                          f'{signal_name} '
-                                          f'{get_value(attribute)};')
-            elif "node" in element:
-                for node_name, node_dict in element['node'].items():
-                    for attribute in node_dict.values():
+    for message in database.messages:
+        for signal in message.signals:
+            if signal.dbc is not None and signal.dbc.attributes_rel is not None:
+                for node_name, node_dict in signal.dbc.attributes_rel.items():
+                    for _, attribute in node_dict.items():
```

Comment:
if you don't use the keys anyways, you can use this:
```suggestion
                    for attribute in node_dict.values():
```